inc/classes/optimization/CSS/class-critical-css.php

```diff
@@ -258,17 +259,18 @@ public function get_public_taxonomies() {
 		$taxonomies = esc_sql( $taxonomies );
 		$taxonomies = "'" . implode( "','", $taxonomies ) . "'";
 
-		$rows = $wpdb->get_results( // WPCS: unprepared SQL ok.
+		$rows = $wpdb->get_results(
 			"
 			SELECT MAX( term_id ) AS ID, taxonomy
 			FROM (
 				SELECT term_id, taxonomy
 				FROM $wpdb->term_taxonomy
-				WHERE taxonomy IN ( $taxonomies )
+				WHERE taxonomy IN ( $taxonomies )" .
+			'
```

Comment:
Same here, why this concat?

Reply:
Fixed here 2cda9aa0fca1d3bd603e4a81bcccfb97bb5a72ce